File: C++/1020.cpp
#include <iostream>
#include <string>
#include <limits>
#include <cmath>
#include <iomanip>

using namespace std;

int main(){

    int days;

    cin >> days;

    int years = days/365;
    days = days - (years*365);

    int months = days/30;
    days = days - (months*30);

    cout << years << " ano(s)\n" << months << " mes(es)\n" << days << " dia(s)\n";


}

File: php/URI/1020.php
<?php

$days = readline("");

$years = intdiv($days, 365);
$days = $days-($years*365);

$months = intdiv($days, 30);
$days = $days - ($months*30);

echo $years ." ano(s)";
echo "\n";
echo $months ." mes(es)";
echo "\n";
echo $days ." dia(s)";
echo "\n";



?>